accumulo/src/main/java/org/vertexium/accumulo/mapreduce/AccumuloEdgeInputFormat.java

```diff
@@ -24,23 +23,27 @@
 import java.util.Map;
 
 public class AccumuloEdgeInputFormat extends AccumuloElementInputFormatBase<Edge> {
-    private static EdgeIterator edgeIterator = new EdgeIterator(AccumuloGraph.toIteratorFetchHints(org.vertexium.FetchHint.ALL));
+    private static EdgeIterator edgeIterator = new EdgeIterator(AccumuloGraph.toIteratorFetchHints(org.vertexium.FetchHint.DEFAULT));
 
     public static void setInputInfo(Job job, AccumuloGraph graph, String instanceName, String zooKeepers, String principal, AuthenticationToken token, String[] authorizations) throws AccumuloSecurityException {
         String tableName = graph.getEdgesTableName();
         setInputInfo(job, instanceName, zooKeepers, principal, token, authorizations, tableName);
     }
 
     @Override
-    protected Edge createElementFromRow(AccumuloGraph graph, PeekingIterator<Map.Entry<Key, Value>> row, Authorizations authorizations) {
+    protected Edge createElementFromRow(
+            AccumuloGraph graph,
+            PeekingIterator<Map.Entry<Key, Value>> row,
+            Authorizations authorizations
+    ) {
         try {
-            EnumSet<FetchHint> fetchHints = AccumuloGraph.toIteratorFetchHints(org.vertexium.FetchHint.ALL);
+            EnumSet<org.vertexium.FetchHint> fetchHints = org.vertexium.FetchHint.DEFAULT;
```

Comment:
There are two `FetchHint` classes. One for iterators and one in Vertexium core. I just renamed the iterator fetch hint to `IteratorFetchHint` and fixed the FQNs